lib/ftp-remote-edit.js

```diff
@@ -1925,7 +1925,7 @@ class FtpRemoteEdit {
 
       self.finderView.on('ftp-remote-edit-finder:open', (item) => {
         let relativePath = item.relativePath;
-        let localPath = normalize(self.finderView.root.getLocalPath() + relativePath, Path.sep);
+        var localPath = normalize(self.finderView.root.getLocalPath() + relativePath, Path.sep);
```

Comment:
Use let instead of var

let localPath = normalize(self.finderView.root.getLocalPath() + relativePath, Path.sep);

Reply:
Thank you for your suggestions and corrections. I will adjust a version as soon as possible. Thank you again